### PR TITLE
all: prepare for import into GerritHub

### PR DESCRIPTION
**Pull request by myitcv**
(see commit message)

### CODE REVIEW

File: internal/ci/workflows.cue [hunk 1]
```diff
@@ -46,134 +46,106 @@ test: _#bashWorkflow & {
...
-	on: ["repository_dispatch"]
-
-	jobs: {
-		test: _#dispatchJob & {
-			_#type:    _#unity
-			strategy:  _#testStrategy
-			"runs-on": "${{ matrix.os }}"
+		delete_build_branch: {
+			"runs-on": _#linuxMachine
+			if:        "${{ \(_#isCLCITestBranch) && always() }}"
```
Comment (mvdan) [inline internal/ci/workflows.cue:110]: interesting that github's syntax highlighting seems to think `if` here is a keyword.

**Review by mvdan — APPROVED** [review]
FWIW I thought that having CUE itself as a go.mod dependency would be okay, but removing it sounds good if we don't actually need to import any Go packages right now.

**Review by mvdan — APPROVED** [review]

**Comment by myitcv** [comment]
> FWIW I thought that having CUE itself as a go.mod dependency would be okay, but removing it sounds good if we don't actually need to import any Go packages right now.

No, you're earlier comment is entirely spot on. We do import `cuelang.org/go/...`, so combining the two makes total sense.

File: internal/ci/workflows.cue (hunk 1 above)
Comment (myitcv) [reply internal/ci/workflows.cue]: Yeh that will be a function of https://github.com/cue-sh/vscode-cue

**Comment by myitcv** [comment]
Rebase-merging this one PR and will close the others

**Pull request merged**